build/main-branch-sync/sync.sh

```
@@ -3,7 +3,7 @@ set -e
 
 path="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 exit_code=0
-CURRENT_VERSION=$(cat ${path}/../../CURRENT_VERSION)
+CURRENT_VERSIONS=$(cat ${path}/../../CURRENT_VERSION)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)